src/Content.php

```diff
@@ -678,12 +678,18 @@ public function getDecodedValue($name)
                     // Parse the field as Markdown, return HTML
                     $value = \ParsedownExtra::instance()->text($value);
 
+                    $config = $this->app['config']->get('general/htmlcleaner');
+                    $allowed_tags = !empty($config['allowed_tags']) ? $config['allowed_tags'] :
```

Comment:
Nah, won't work. 

The way our stupid getdefaults works with regard to merging, prevents this.. For example, if i set in config:

```
htmlcleaner:
  allowed_tags: [ x, y, z] 
```

And the default is `a, b, c, d, e, f`, you would end up with `x, y, z, d, e, f`. Not what somebody would want. 

(this had me puzzled for a while, because i didn't understand why this was happening ;-) )